File: java/source/com/nehu/factor/funfactor.java
package source.com.nehu.factor;

public class funfactor {

  public static void main(String[] args) {
    factor();
  }

  static void factor() {
    int n = 60;

    for (int i = 1; i <= n; i++) {
      if (n % i == 0) {
        System.out.println(i);
      }
    }
  }
}

File: c++/funfactor.cpp
#include <iostream>
using namespace std;
void factor();
int main()
{
    factor();
}
void factor()
{
    int n=60, i;

    for (int i = 1; i <= n; i++)
    {
        if (n%i==0)
       {
        cout<<i<<" ";
       }
    }  
}
